### PR TITLE
Enable users to register multisignature groups - Closes 3136

### LINKED ISSUE #3136: Use Service API endpoints to register multisig account
### Description
For the sake of having a foundation to work with, the current multisignature account registration form works with mock values. We should connect the view to Service API endpoints to actually register an account.

### Motivation
Make it real!

### Acceptance Criteria
- Rewire the registration button to read data from the account.
- Update transactionBroadcasted action to accept this tx type.
- Update fee calculation to accept this tx type.
- Update the form to enable users to choose processing speed.

### PR DESCRIPTION
**Pull request by UsamaHameed**
### What was the problem?
This PR resolves #3136

### CODE REVIEW

File: src/components/screens/multiSignature/editor/editor.js [hunk 1]
```diff
@@ -144,10 +185,12 @@ const Editor = ({
...
           selectedPriority={selectedPriority.selectedIndex}
           setSelectedPriority={selectTransactionPriority}
+          loadError={prioritiesLoadError}
+          isLoading={loadingPriorities}
         />
-        {
+        {/* {
           feedback.error && <span className="feedback">{feedback.messages[0]}</span>
-        }
+        } */}
```
Comment (isalga) [inline src/components/screens/multiSignature/editor/editor.js:193]: Should this not be uncommented?

File: src/components/screens/multiSignature/editor/editor.js [hunk 2]
```diff
@@ -13,60 +13,77 @@ import MemberField from './memberField';
...
+
+    return [mandatory, optional];
+  }, [members]);
+
   const { fee, minFee } = useTransactionFeeCalculation({
     selectedPriority,
     token,
     account,
     priorityOptions,
+    // numberOfSignatures: requiredSignatures,
```
Comment (isalga) [inline src/components/screens/multiSignature/editor/editor.js:68]: Why is it commented? If it is not needed why not to remove it?

File: src/components/screens/multiSignature/result/result.js [hunk 3]
```diff
@@ -8,34 +8,22 @@ import ProgressBar from '../progressBar';
...
-
-      if (confirmed.length) setStatus('ok');
-      if (broadcastError.length) setStatus('fail');
-    }
-  }, [transactions]);
-
-  useEffect(() => {
-    if (transactionInfo) transactionBroadcasted(transactionInfo);
-  }, [transactionInfo]);
+  const onDownload = () => {
```
Comment (isalga) [inline src/components/screens/multiSignature/result/result.js:21]: We already had the same function on another file. maybe lets move to utils and import it from there

File: src/components/screens/multiSignature/editor/editor.js (hunk 1 above)
Comment (UsamaHameed) [reply src/components/screens/multiSignature/editor/editor.js:193]: I will work on the validation in a separate ticket.

**Review by isalga — APPROVED** [review]

**Pull request merged**